pkg/asset/machines/master.go

```diff
@@ -359,6 +359,9 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		mpool.Set(pool.Platform.BareMetal)
 		pool.Platform.BareMetal = &mpool
 
+		// Use managed user data secret, since we always have up to date images
+		// available in the cluster
+		masterUserDataSecretName = "master-user-data-managed"
```

Comment:
Is this change a requirement for deploying the CoreOS ISO from the release payload? Or was it just convenient to include this change in this PR?

Reply:
Fair enough. I was just checking because the intricacies of these changes are a bit too much for me on a quick work through the PR.